docs/internal-documentation/maintenance/ReleaseProcess-create-test-distribution.md

```diff
@@ -5,7 +5,13 @@ The purpose of this part is to create the test distributions that can be
 used in QA testing for a release. It can also be used to create test
 distributions to verify a pull request.
 
-## 1. Create a clean environment
+
+## 1. Prepare a build environment
+Set up build system to be used for the test distribution creation. See
+[Build Environment Preparation] for set it up.
```

Comment:
`for set it up` -> I would say `for set up` or `to set it up` (but I'm not 100% sure)

Reply:
Fixed.